### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by nohjlau**
Addresses #15 

Updated the sidebar so that the inactive entries display the title and thumbnail and the active entry displays all of the information and shows a full image while hiding the thumbnail.

**Inactive entry**
image

**Active entry**
image

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -133,36 +133,52 @@ a:hover {
   padding: 0 10px;
 }
 
-.listings {
+#listings {
```
Comment (JMStudiosJoe) [inline css/main.css]: why the change from class to ID?

Comment (nohjlau) [reply css/main.css]: I changed the listings element from a class to an id because it's part of the layout, so we'll only have one listings element which we populate with many entry class elements.
